pkg/restore/client.go

```diff
@@ -181,7 +181,7 @@ func (rc *Client) InitBackupMeta(c context.Context, backupMeta *backuppb.BackupM
 
 		var ddlJobs []*model.Job
 		// ddls is the bytes of json.Marshal
-		ddls, err := reader.ReadDDLs(c)
+		ddls, _ := reader.ReadDDLs(c)
```

Comment:
shall we check insted of ingore of ignore this error? @3pointer

Reply:
We should check the error.